src/app/hooks/use-deeplink.test.tsx

```diff
@@ -116,10 +116,7 @@ describe("useDeeplink hook", () => {
 
 	it("should subscribe to deeplink listener and navigate when no method found", () => {
 		ipcRenderer.on.mockImplementationOnce((event, callback) =>
-			callback(
-				event,
-				"ark:vote?coin=ark&network=mainnet&recipient=DNjuJEDQkhrJ7cA9FZ2iVXt5anYiM8Jtc9&amount=1.2&memo=ARK&delegate=alessio",
-			),
+			callback(event, "ark:vote?coin=ark&network=mainnet&delegate=alessio"),
```

Comment:
```suggestion
			callback(event, "ark:vote?coin=ark&network=ark.mainnet&delegate=alessio"),
```

The networks are prefixed with the specific coin name because you could have the parent coin `ark` but then networks like `compendia.mainnet` instead of just mainnet for `ark`.